android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java

```diff
@@ -43,7 +43,7 @@
   public static final String ARG_URL = "KMKeyboardActivity.url";
   public static final String ARG_FILENAME = "KMKeyboardActivity.filename";
 
-  public static final String kKeymanApiBaseURL = "https://r.keymanweb.com/api/3.0/";
+  public static final String kKeymanApiBaseURL = "https://api.keyman.com/cloud/3.0/";
   public static final String kKeymanApiRemoteURL = "https://r.keymanweb.com/api/2.0/remote?url=";
```

Comment:
Does `kKeymanApiRemoteURL` get changed too?

Reply:
No, that doesn't change because it's a different API. We'll deal with that one separately (by eliminating it ...) It's a proxying service for ad-hoc keyboards which was a grand idea but ultimately unnecessary.